lib/brains/models.py

```diff
@@ -69,3 +70,12 @@ class Meta(Model.Meta):
     def get_uri(self):
         return reverse('braintree:mozilla:subscription-detail',
                        kwargs={'pk': self.pk})
+
+    def get_transactions(self, ids=None):
+        query = (Transaction.objects
+                 .filter(buyer=self.paymethod.braintree_buyer.buyer,
+                         seller_product=self.seller_product))
+        if ids:
+            query = query.filter(id__in=ids)
+
+        return query.order_by('-created')
```

Comment:
I'm not sure this is worth putting on the model, tempted to move it back to webhooks.